pennylane/operation.py

```diff
@@ -799,6 +799,68 @@ def eigvals(self):
     def _matrix(cls, *params):
         return np.diag(cls._eigvals(*params))
 
+class Channel(Operation):
```

Comment:
Small comment here: to me, `Operation` would be unitary. If there are many functionalities in the `Operation` class which would be copied, then maybe it's worth leaving as is and breaking with the convention of unitarity. Otherwise, subclassing `Operator` could also be considered. (Also feel free to ignore this comment if this was discussed in details previously.)

Reply:
I actually had the same question but was not sure.  I am only using ``grad_method`` from the operation class. Is operation unitary due to the ``inverse`` and ``decomposition`` property? it is not explicitly stated anywhere that all operations will be unitary.

Reply:
I feel there are many extra methods even with ``Operator`` that are not required.

Reply:
Should we make a new abstract class of its own?

Reply:
Yeah, found [this occurrence](https://github.com/PennyLaneAI/pennylane/blob/d6ed14b45f7fa86651118c423d45bc4d74f38bd6/pennylane/operation.py#L39) which mentions a "unitary quantum gate" as an example for an `Operation`. Overall happy to have `Channel` subclass `Operation`, if we decide to do so. Overall might just come down to further consideration.

Reply:
Thanks Antal. We definitely want to make sure to catch all required changes to the documentation. Adding channels as operation was already discussed in the [ADR](url) and it is one of the first decisions that was reached.

Reply:
(fly-by-comment: we always envisioned the `Operation` class representing non-unitary operations, but never quite got there 🙂 Hence why the docstring only gives unitary examples.

Note that PennyLane is always a WIP, even code that has been unchanged for a while! So it is always fine to update existing code/docstrings in new PRs, rather than restrict our changes to suit the existing code/docstrings.)